roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -152,15 +152,20 @@ spec:
 {% if fjelltopp_cloud_provider != 'azure' %}
             - name: AWS_DEFAULT_REGION
               value: "{{ aws_region }}"
+            - name: CKAN_SAML_IDP_CERT
+              value: |-
+                {{ ckan_saml_idp_cert }}
```

Comment:
This will always expect to see the cert... even for projects not using SSO, no?

Reply:
The variable will just be empty 🤷

Reply:
Did you do a clean local deploy of something not using saml? Experience (e.g. things like Google analytics) say this will break

Reply:
of course you were right. added initialisation of an empty variable and retested with success on afro local